src/governance.rst

```diff
@@ -144,6 +144,24 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
+    create a poll with an external tool or call
+    for voting on the GH issue in question.
+    The voting must period be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions.
+
+    * Standard
+    * 50% Majority to pass
+
+:Modifying the governance document: The voting should happen in the PR
+    in question. The voting must period be open for at least one core
```

Comment:
"The voting period must..."?

Reply:
Maybe the PR also needs to have a `@conda-forge/core` call somewhere?

Reply:
Should we put this governance in its own repo so we can have review requirements?

Reply:
> Maybe the PR also needs to have a @conda-forge/core call somewhere?